packages/framework/src/Migrations/Version20190521071335.php

```diff
@@ -0,0 +1,27 @@
+<?php
+
+namespace Shopsys\FrameworkBundle\Migrations;
+
+use Doctrine\DBAL\Schema\Schema;
+use Shopsys\MigrationBundle\Component\Doctrine\Migrations\AbstractMigration;
+
+class Version20190521071335 extends AbstractMigration
+{
+    /**
+     * @param \Doctrine\DBAL\Schema\Schema $schema
+     */
+    public function up(Schema $schema)
+    {
+        $this->sql('ALTER TABLE products ADD uuid UUID DEFAULT NULL');
+        $this->sql('UPDATE products SET uuid = uuid_generate_v1()');
```

Comment:
The decision was based on https://blog.programster.org/mysql-performance-when-using-uuid-for-primary-key

I know it's MySQL, but the point is the same - UUID1 is less random, and therefore should produce less spread values.
Also external systems should use UUID1 for import to prevent collisions

UUID4 is used in Product only as a fallback - if noone provides a reliable UUID, it creates it's own...

But when I take a look into the database, UUID1 are not less random than UUID4; If the external system will use UUID4/UUID1, the outcome will be sort of the same; and if the external system will use poor random number generator, then the import will just fail.

Chaging to UUID4 as it is consistent and without questions (and UUID1 doesn't seem to add a value)